spring-boot-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -20,4 +20,7 @@ org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
   org.axonframework.springboot.autoconfig.AxonServerActuatorAutoConfiguration,\
   org.axonframework.springboot.autoconfig.AxonServerBusAutoConfiguration, \
   org.axonframework.springboot.autoconfig.AxonTracingAutoConfiguration, \
-  org.axonframework.springboot.autoconfig.OpenTelemetryAutoConfiguration
\ No newline at end of file
+  org.axonframework.springboot.autoconfig.OpenTelemetryAutoConfiguration
+org.springframework.boot.autoconfigure.service.connection.ConnectionDetailsFactory=\
```

Comment:
That's curious! Didn't expect these connection-specific classes to require a change to the `spring.factories` document.
Especially since this document was introduced for SB1 and SB2 autoconfiguration support, and that we had to introduce a different file for SB3 autoconfiguration support.

Reply:
Now that I'm on the subject of SB2 to SB3 difference, perhaps it's good to mark the new connection details logic as SB3 and up specific support in the JavaDoc?